crates/re_query_cache/src/cache.rs

```diff
@@ -770,6 +770,7 @@ impl CacheBucket {
         Ok(added_size_bytes)
     }
 
+    /// This will insert an empty slice for a missing component (instead of N `None` values).
```

Comment:
Why do we need to insert/store anything? Can't we just return an empty container from the cache if the component-entry is missing when we try to access it?

Reply:
We need to keep track of the empty rows so all the component caches can share the same top-level offsets (similar to what we do in the store where we use a `Option::<DataCell>::None` value for that).

Reply:
Ahh, for the case of range-queries where only some of the rows are empty?